src/main/java/com/google/sps/servlets/YouTubeServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.List;
+import java.util.Arrays;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.stream.Collectors;
+import com.google.sps.data.Video;
+
+@WebServlet("/yt")
```

Comment:
Nit: Servlet method should be representative of what it does (just like method names).